aur/repos2aur

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if (( $# == 0 )); then
+  echo "error: No arguments passed."
+  echo "pkgbase is a required argument"
+  exit 1
+fi
+
+PKGBASE=$1
+
+if [[ ! -d "$PKGBASE" ]]; then
+  echo "error: directory $1 does not exists"
+  exit 1
+fi
+
+TMPDIR=$(mktemp -d)
+
+function cleanup {
+  rm -rf "$TMPDIR"
+}
+
+trap cleanup EXIT
+
+
+git clone "ssh://aur@aur.archlinux.org/$PKGBASE.git" $TMPDIR
+cp $PKGBASE/trunk/* $TMPDIR
```

Comment:
Hmm, if $PKGBASE is expected to exist already, then it might contain .pkg.tar* files left over. Do we care about that?

Reply:
Hm good point, but not sure how we would tackle those files some glob?

Reply:
`svn export` would be one option. Another would be teaching it to accept a pkgbase rather than a directory name, and doing the svn checkout in the script itself rather than doing it manually beforehand.

Reply:
created an svn export solution